containers/unit_tests/TestWithoutInitializing.hpp

```diff
@@ -37,6 +37,16 @@
 #endif
 ///@}
 
+/// Some tests are skipped for unified memory space.
+///@{
```

Comment:
Wasn't that comment syntax intended for the `@todo` in the CUDA UVM block above which is not applicable to what you are adding?

Reply:
What do I know about random syntax lol

Reply:
done